### DISCUSSION TITLE
Unable to change Nextcloud AIO (Linux) PHP Memory and Upload Limit

### DISCUSSION THREAD

**Discussion by SophyInBetween** [nextcloud/all-in-one#3033] [Questions] [answered]
Hello, I am using a AIO Installation on an virtual ubuntu server, 

I did stop the mastercontainer (sudo docker stop nextcloud-aio-mastercontainer ) and removed it (sudo docker rm nextcloud-aio-mastercontainer) then double checked in the list of docker containers, it was really gone and then tried the following commands: 

# For Linux and without a web server or reverse proxy (like Apache, Nginx, Cloudflare Tunnel and else) already in place:
```console
sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
--env NEXTCLOUD_UPLOAD_LIMIT=150G
--env NEXTCLOUD_MAX_TIME=64800
--env NEXTCLOUD_MEMORY_LIMIT=3072M
nextcloud/all-in-one:latest

```
As mentioned in the manual. 

i got the following: 

"docker run" requires at least 1 argument.
See 'docker run --help'.

Usage:  docker run [OPTIONS] IMAGE [COMMAND] [ARG...]

Create and run a new container from an image
--env: command not found
--env: command not found
-bash: nextcloud/all-in-one:latest: No such file or directory

without the environmet variables it works fine and my mastercontainer is up and running, but i really like to change the upload-limit, upload-time and php memory. 

I am currently running [Nextcloud Hub 4](https://nextcloud.com/) (26.0.3) and according to the aio-interface all containers are up-to-date.

**Comment by Zoey2936** [comment, marked as answer]
you forgot the backslash at the end of the lines, as you can see

**Reply by SophyInBetween** [reply]
well, thanks, really.
